toolchain/power_time_base_bug.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2018 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+import shutil
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import distro
+from avocado.utils.software_manager import SoftwareManager
+
+
+class PowerTimeBaseBug(Test):
+    '''
+    PowerTimeBaseBug is based on valgrind. Valgrind is truncating
+    output of __ppc_get_timebase value to 32 bit.
+
+    This test case verify whether we have bug exist or not.
+
+    :avocado: tags=toolchain,ppc64le
+    '''
+
+    def setUp(self):
+        '''
+        Check for required packages namely gcc, make, valgrind.
+        Transfer the source file and make file.
+        Compile
+        '''
+        smm = SoftwareManager()
+        self.failures = []
+
+        dist = distro.detect()
```

Comment:
Looks to be unused. We can add a check to skip if `arch` is not `ppc64le`

Reply:
@harish-24 Sure.